Sources/PackageGraph/ModuleAliasTracker.swift

```diff
@@ -141,6 +141,8 @@ class ModuleAliasTracker {
         fillInRest(package: rootPkg)
     }
 
+    var emittedDiagnosticsFor = [String]()
```

Comment:
this is needed because otherwise we emit the diagnostic twice. This feels like a bug on its own because we shouldn't need to process the same module alias twice?